index.html

```diff
@@ -1,105 +1,166 @@
-                            <i class="fa fa-youtube" aria-hidden="true"></i>
-                            <span class="sr-only">YouTube</span>
-                        </a>
-                    </li>
-                </ul>
+            <div class="carousel-item">
+              <img src="assets/media/palm-rock.webp" class="d-block w-100" alt="palm tree and rock on beach">
             </div>
-            <div class="col-sm-3">
-                <p>Payment types accepted</p>
-                <ul class="list-inline payment-icons">
-                  <li class="list-inline-item">
-                    <i class="fa-brands fa-paypal"></i>
-                    <span class="sr-only">Paypal</span>
-                  </li>
-                  <li class="list-inline-item">
-                    <i class="fa-brands fa-cc-amex"></i>
-                    <span class="sr-only">American Express</span>
-                  </li>
-                  <li class="list-inline-item">
-                    <i class="fa-brands fa-google-pay"></i>
-                    <span class="sr-only">Google Pay</span>
-                  </li>
-                  <li class="list-inline-item">
-                    <i class="fa-brands fa-apple-pay"></i>
-                    <span class="sr-only">Apple Pay</span>
-                  </li>
-                </ul>
+            <div class="carousel-item">
+              <img src="assets/media/pina-coco.jpg" class="d-block w-100" alt="pineapple and coconut at the beach">
             </div>
+            <div class="carousel-item">
+              <img src="assets/media/suitcases.avif" class="d-block w-100" alt="pastel coloured suitcases">
+            </div>
+            <div class="carousel-item">
+              <img src="assets/media/vintage-suitcases.jpg" class="d-block w-100" alt="a stack of vinatge suitcases">
+            </div>
+            <div class="carousel-item">
+              <img src="assets/media/water-huts.avif" class="d-block w-100" alt="beach-front huts">
+            </div>
+          </div>
+          <button class="carousel-control-prev" type="button" data-bs-target="#carouselRide" data-bs-slide="prev">
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="visually-hidden">Previous</span>
+          </button>
+          <button class="carousel-control-next" type="button" data-bs-target="#carouselRide" data-bs-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="visually-hidden">Next</span>
+          </button>
         </div>
-    </footer>
+      </div>
+      <div class="col-2"></div>
+    </div>
 
-<!-- Bootstrap for Javascript -->
+    <!-- About section -->
 
+    <div class="row about">
+      <div class="col-12">
+        <h3><u>All About Travel Away</u></h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam in nobis blanditiis quibusdam eos mollitia et
```

Comment:
Do you intend to replace the Lorem ipsum text in the about section, or leave it as is?